plugin/web/extensions/location/modals/commonModalWrapper/index.tsx

```diff
@@ -0,0 +1,114 @@
+import { Divider, Button, Icon } from "@web/sharedComponents";
+import { styled } from "@web/theme";
+import { Children, Fragment, ReactNode } from "react";
+
+export type Props = {
+  title?: string;
+  children?: ReactNode;
+  onModalChange: () => void;
+};
+
+const CommonModalWrapper: React.FC<Props> = ({ title, children, onModalChange }) => {
+  const childArray = Children.toArray(children);
+
+  return (
+    <Wrapper>
+      {title && (
+        <>
+          <HeaderWrapper>
+            <Title>{title}</Title>
+            <CloseButton>
+              <Icon icon="close" onClick={onModalChange} size={16} />
+            </CloseButton>
+          </HeaderWrapper>
+          <Divider />
+        </>
+      )}
+      {childArray.map((child, idx) => (
+        <FragmentWrapper key={idx}>
+          {child}
+          {idx + 1 !== childArray.length && <span />}
```

Comment:
Is this just to try and make some extra space at the bottom? If so, please use css on the Wrapper (like last child or whatever it is called, I forget...)

Reply:
css changes